motoman_driver/MotoPlus/Controller.c

```diff
@@ -515,6 +446,8 @@ void Ros_Controller_StatusInit(Controller* controller)
 	controller->ioStatusAddr[IO_ROBOTSTATUS_ESTOP_CTRL].ulAddr = 80027;   		// Controller E-Stop
 	controller->ioStatusAddr[IO_ROBOTSTATUS_WAITING_ROS].ulAddr = IO_FEEDBACK_WAITING_MP_INCMOVE; // Job input signaling ready for external motion
 	controller->ioStatusAddr[IO_ROBOTSTATUS_INECOMODE].ulAddr = 50727;			// Energy Saving Mode
+	controller->ioStatusAddr[IO_ROBOTSTATUS_PFL_STOP].ulAddr = 81702;			// PFL function stopped the motion
+	controller->ioStatusAddr[IO_ROBOTSTATUS_PFL_ESCAPE].ulAddr = 81703;			// PFL function escape from clamping motion

```

Comment:
There is no updated `IONAME.DAT` in this PR, is that something we'd want to add, or do these IOs already have names/comments associated with them (note: I don't have an HC-10 or FSU, so this may be a stupid question).

Reply:
The 8xxxx series are "internal control status" signals. These are controlled by internal controller software and cannot be named by the user.